### ISSUE TITLE
Dual booting?

### ISSUE THREAD

**Issue by Neglectfull** [Vanilla-OS/vanilla-installer#162]
Is it possible to dual boot this os?

**Comment by MonsterObserver** [comment]
Yes it is. You will need to change some GRUB settings for GRUB to show your other OS, if you need help with that reopen the issue :D

**Comment by Neglectfull** [comment]
Yeah, thanks! I'm a beginner tho.. so it would be great if you just told me what I need to change. Thank you for the help!

**Comment by kbdharun** [comment]
> Yeah, thanks! I'm a beginner tho.. so it would be great if you just told me what I need to change. Thank you for the help!

Feel free to join our Discord Server if you haven't joined already we have this pinned in our #support channel.

TLDR

## Minimum Requirements

- 1GB FAT partition for EFI (If legacy systems, 1 MiB for BIOS partition. 
The filesystem type is hardcoded and can't be changed in the installer).
- 1 GB ext4 partition for boot (the filesystem type is hardcoded and can't be changed in the installer).
- 2 10GB+ (Recommend 15/20 GB ideally) root partitions as either ext4 or btrfs.
- 1 home partition in whatever size and filesystem type you like, we recommend btrfs.

## Steps

- Add `GRUB_DISABLE_OS_PROBER=false` to `/etc/default/grub`, then start an empty transaction with `sudo abroot exec echo`, then `reboot` and press ESC key right after startup to show grub.
> GRUB is hidden by default to unhide it follow the below steps
- Change
```
GRUB_TIMEOUT=0
GRUB_HIDDEN_TIMEOUT=2
GRUB_TIMEOUT_STYLE=hidden
```
to
```
GRUB_TIMEOUT=2
GRUB_HIDDEN_TIMEOUT=2
GRUB_TIMEOUT_STYLE=menu
```

**Comment by Neglectfull** [comment]
> > Yeah, thanks! I'm a beginner tho.. so it would be great if you just told me what I need to change. Thank you for the help!
> 
> Feel free to join our Discord Server if you haven't joined already we have these steps pinned in our #support channel.
> 
> TLDR
> 
> ## Minimum Requirements
> 
> - 1GB FAT partition for EFI (If legacy systems, 1 MiB for BIOS partition. 
> The filesystem type is hardcoded and can't be changed in the installer).
> - 1 GB ext4 partition for boot (the filesystem type is hardcoded and can't be changed in the installer).
> - 2 10GB+ (Recommend 15/20 GB ideally) root partitions as either ext4 or btrfs.
> - 1 home partition in whatever size and filesystem type you like, we recommend btrfs.
> 
> ## Steps
> 
> - Add `GRUB_DISABLE_OS_PROBER=false` to `/etc/default/grub`, then start an empty transaction with `sudo abroot exec echo`, then `reboot` and press ESC key right after startup to show grub.
> > GRUB is hidden by default to unhide it follow the below steps
> - Change this inside `/etc/default/grub` and start an empty again with `sudo abroot exec echo`
> ```
> GRUB_TIMEOUT=0
> GRUB_HIDDEN_TIMEOUT=2
> GRUB_TIMEOUT_STYLE=hidden
> ```
> to
> ```
> GRUB_TIMEOUT=2
> GRUB_HIDDEN_TIMEOUT=2
> GRUB_TIMEOUT_STYLE=menu
> ```
> - Now `reboot` and you should see your other OS and GRUB automatically
> 

Thanks :) it's pretty godamn complicated... But I'll try it and see how it goes! Thank you a ton! :)

**Comment by orowith2os** [comment]
@kbdharun 

> Feel free to join our Discord Server if you haven't joined already we have these steps pinned in our #support channel.

It's not good to reference a Discord channel on public issues such as this, it should be available on public docs instead, and you reference that here. If you find yourself needing to say this, you should look into adding it to the docs.
